docs/admin/Metadata/index.rst

```diff
@@ -0,0 +1,115 @@
+.. admin_Metadata_index:
+
+.. _`audittrail` : ../Logging/index.hmtl
+.. _`Wet open overheid` : https://wetten.overheid.nl/BWBR0045754/
+.. _`Woo-index`: https://open.overheid.nl/
+.. _`DiWoo` : https://standaarden.overheid.nl/diwoo/metadata
+.. _`Woo informatiecategorieën` : https://standaarden.overheid.nl/tooi/waardelijsten/work?work_uri=https%3A%2F%2Fidentifier.overheid.nl%2Ftooi%2Fset%2Fscw_woo_informatiecategorieen
+.. _`Thema-indeling voor Officiële Publicaties (TOP-lijst)` : https://standaarden.overheid.nl/tooi/waardelijsten/work?work_uri=https%3A%2F%2Fidentifier.overheid.nl%2Ftooi%2Fset%2Fscw_toplijst
+.. _`audittrail` : ../Logging/index.hmtl
+
+Metadata
+============
+
+Onder het menu-item "Metadata" en op het dashboard onder het kopje "Metadata" wordt toegang geboden tot het beheer van:
+
+* Informatiecategorieën
+* Thema's
+
+Door herop te klikken wordt het desbetreffende beheerscherm geopend.
+
+Informatiecategorieën
+------------------------
+
+Toelichting
+~~~~~~~~~~~~~
+
+De `Wet open overheid`_ beschrijft 17 *informatiecategorieën*. Documenten die onder minstens één van deze categorieën vallen, moeten *actief* openbaar gemaakt worden. Dat wil zeggen dat een bestuursorgaan deze online moet publiceren, ook wanneer daar niet door een burger expliciet om gevraagd is in een *Woo-verzoek*. 
+
+Daarnaast vraagt de wet om ook documenten openbaar te maken die buiten deze *informatiecategorieën* vallen. Hiervoor geldt geen "harde" plicht, maar wel een *inspanningsverplichting*.
+
+Overigens gelden op de plicht tot openbaarmaking ook een aantal uitzonderingen (hoofdstuk 5 van de wet). Raadpleeg voor meer informatie over de wet de jurist en/of informatiebeheerder van uw eigen organisatie.
```

Comment:
```suggestion
Overigens gelden op de plicht tot openbaarmaking ook een aantal uitzonderingen (hoofdstuk 5 van de wet). Raadpleeg voor meer informatie over de wet de jurist en/of informatiebeheerder van je eigen organisatie.
```
recent developments show that information language (in user interfaces, documentation...) is preferred